plone/app/content/browser/contents/__init__.py

```diff
@@ -146,26 +146,43 @@ def get_actions(self):
     def get_columns(self):
         # Base set of columns
         columns = {
-            'id': translate(_('ID'), context=self.request),
-            'ModificationDate': translate(_('Last modified'), context=self.request),  # noqa
-            'EffectiveDate': translate(_('Publication date'), context=self.request),  # noqa
-            'ExpirationDate': translate(_('Expiration date'), context=self.request),  #noqa
             'CreationDate': translate(_('Created on'), context=self.request),  # noqa
-            'review_state': translate(_('Review state'), context=self.request),  # noqa
-            'Subject': translate(_('Tags'), context=self.request),
-            'portal_type': translate(_('Type'), context=self.request),
-            'is_folderish': translate(_('Folder'), context=self.request),
+            'Creator': translate(_('Creator'), context=self.request),
+            'Description': translate(_('Description'), context=self.request),
```

Comment:
This *might* be problematic in cases when ``Description`` is a lot of text. Then including ``Description`` would raise the payload a lot. Can be fixed if - e.g. the structure pattern - only requests the data it really needs (currently it requests all data, which can be shown in folder_listing columns).